### PR TITLE
Manipulation rare: Feast with placeholder art.

### PR DESCRIPTION
**Pull request by Frenoss**

### CODE REVIEW

File: src/main/java/stsjorbsmod/cards/Feast.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+        tags.add(REMEMBER_MEMORY);
+    }
+
+    @Override
+    public void upgrade() {
+        if (!upgraded) {
+            upgradeName();
+            upgradeMagicNumber(UPGRADE_PLUS_DRAW);
+            //initializeDescription();
```
Comment (dbjorge) [inline src/main/java/stsjorbsmod/cards/Feast.java]: Is this intended to be commented out?

Comment (Frenoss) [reply src/main/java/stsjorbsmod/cards/Feast.java]: I'll remove it for now. It seems like most upgrade() overrides don't need to do anything extra to their description, though they do.

File: src/main/java/stsjorbsmod/cards/Feast.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        magicNumber = baseMagicNumber = DRAW;
+        exhaust = true;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        enqueueAction(new RememberSpecificMemoryAction(new GluttonyMemory(p, false)));
+        enqueueAction(new DrawCardAction(p, magicNumber, false));
+
+        tags.add(REMEMBER_MEMORY);
```
Comment (dbjorge) [inline src/main/java/stsjorbsmod/cards/Feast.java]: I think the tag addition needs to go in the ctor, not `use` (it exists so `Weight of Memory` can find the card)

Comment (Frenoss) [reply src/main/java/stsjorbsmod/cards/Feast.java]: I'll change this here. Does that mean MirrorImage, for instance, should be fixed the same way?

Comment (dbjorge) [reply src/main/java/stsjorbsmod/cards/Feast.java]: Yep, oops

**Pull request merged**